README.md

```diff
@@ -145,15 +146,15 @@ There are implementations for the following logging libraries:
 ## slog interoperability
 
 Interoperability goes both ways, using the `logr.Logger` API with a `slog.Handler`
-and using the `slog.Logger` API with a `logr.LogSink`. [slogr](./slogr) provides `NewLogr` and
-`NewSlogHandler` API calls to convert between a `logr.Logger` and a `slog.Handler`.
+and using the `slog.Logger` API with a `logr.LogSink`. `NewFromSlogHandler` and
```

Comment:
ISTR an earlier draft of this used names like `ToSlog` and `FromSlog` or `{To,From}SlogHandler` - now that I have come around, do you want to reconsider those?